<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOnPlotDialog</class>
 <widget class="QDialog" name="AddOnPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Число последних отсчетов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_MaxLastN">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Подпись под X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xTitleEdit"/>
       </item>
      </layout>
     </item>
     <item row="19" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Единица измерения по X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xUnitComboBox"/>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Подпись под Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yTitleEdit"/>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Физическая величина Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yPhysQuantCbBox"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xNameComboBox"/>
       </item>
      </layout>
     </item>
     <item row="15" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Шкала X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xScaleTypeCbBox">
         <item>
          <property name="text">
           <string>Линейная</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Логарифмическая</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mainLabel">
       <property name="text">
        <string>Добавить на график</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Максимальное число отсчетов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_MaxN">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>3000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yNameComboBox"/>
       </item>
      </layout>
     </item>
     <item row="18" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Цвет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorComboBox"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Направление X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xDirComboBox">
         <item>
          <property name="text">
           <string>Вправо</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Влево</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Метод обновления</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="updateModeCbBox">
         <item>
          <property name="text">
           <string>Отображать все</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Последние N отсчетов</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="20" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Единица измерения по Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yUnitComboBox"/>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Физическая величина X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="xPhysQuantCbBox"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="graphNameLabel">
         <property name="text">
          <string>Имя графика</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="graphNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Направление Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yDirComboBox">
         <item>
          <property name="text">
           <string>Вверх</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Вниз</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="16" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Шкала Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yScaleTypeCbBox">
         <item>
          <property name="text">
           <string>Линейная</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Логарифмическая</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="customPlotNameLabel">
         <property name="text">
          <string>Имя окна</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customPltNameLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddOnPlotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOnPlotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
